bom/public/pom.xml

```diff
@@ -84,7 +84,7 @@
             </dependency>
             <dependency>
                 <groupId>org.hibernate.search</groupId>
-                <artifactId>hibernate-search-mapper-orm-coordination-outbox-polling</artifactId>
+                <artifactId>hibernate-search-mapper-orm-outbox-polling</artifactId>
                 <version>${project.version}</version>
             </dependency>
```

Comment:
Maybe relocation artifacts should be present in the BOM... WDYT?
Also, what about the JSR-352 ones?